src/tools/build-manifest/src/main.rs

```diff
@@ -514,15 +514,6 @@ impl Builder {
             components.push(host_component("rust-mingw"));
         }
 
-        // The compiler libraries are not stable for end users, but `rustc-dev` was only recently
-        // split out of `rust-std`. We'll include it by default as a transition for nightly users,
-        // but ship it as an optional component on the beta and stable channels.
-        if self.rust_release == "nightly" {
-            components.push(host_component("rustc-dev"));
-        } else {
-            extensions.push(host_component("rustc-dev"));
```

Comment:
hm, shouldn't we always push rustc-dev into extensions?

Reply:
Yes, see below where I removed the `target != host` filter. Before, the host was conditional here, and now they all go in extensions.